ultraviolet/shared/src/main/scala/ultraviolet/macros/ShaderMacroUtils.scala

```diff
@@ -30,6 +31,7 @@ trait ShaderMacroUtils:
     case v: ShaderAST.For                     => ShaderAST.unknownType
     case v: ShaderAST.Switch                  => ShaderAST.unknownType
     case v: ShaderAST.Val                     => findReturnType(v.value)
+    case v: ShaderAST.MultiStatements         => findReturnType(v.statements.head)
```

Comment:
Can the statements list ever be empty?

Reply:
No, the list is non-empty.
Since `MultiStatements` is only built from Tuple2-4.